e2e-tests/portal/cypress/support/portal/createBssEwcsDeal.js

```diff
@@ -0,0 +1,178 @@
+import { add } from 'date-fns';
+
+const {
+  dashboard,
+  selectScheme,
+  beforeYouStart,
+  bankDetails,
+  contract,
+  eligibilityCriteria,
+  bondDetails,
+  bondFinancialDetails,
+  bondFeeDetails,
+  contractReadyForReview,
+  contractAboutSupplier,
+  contractAboutBuyer,
+  contractAboutFinancial,
+  eligibilityDocumentation,
+} = require('../../e2e/pages');
+const { submissionDetails } = require('../../fixtures/deal');
+const MOCK_USERS = require('../../../../e2e-fixtures');
+
+const { BANK1_MAKER1 } = MOCK_USERS;
+
+const tomorrow = add(new Date(), { days: 1 });
+const startDate = {
+  day: tomorrow.getDate(),
+  month: tomorrow.getMonth(),
+  year: tomorrow.getFullYear(),
+};
+
+const endDate = {
+  day: tomorrow.getDate(),
+  month: tomorrow.getMonth(),
+  year: tomorrow.getFullYear() + 1,
+};
+
+/**
+ * Create a BSS/EWCS deal via the UI.
+ * @param {Boolean} readyForCheck: Conditionally complete all "maker" required forms
+ */
+const createBssEwcsDeal = ({ readyForCheck = false, dealType, facilityStage, exporterCompanyName }) => {
```

Comment:
`dealType` is very confusing - this should be `dealSubmissionType`

FWIW by adding `= {}` this can be called with `cy.createBssEwcsDeal()` without a parameter which is a bit nicer 

```suggestion
const createBssEwcsDeal = ({ readyForCheck = false, dealSubmissionType, facilityStage, exporterCompanyName } = {}) => {
```

Reply:
Changed as suggested